docs/userGuide/plugins/mermaid.md

```diff
@@ -0,0 +1,96 @@
+### Plugin: Mermaid
+
+This plugin allows you to utilize [Mermaid](https://mermaid-js.github.io/mermaid/) by automatically importing the library and initializing the rendering of the diagrams.
+
+> Mermaid is a JavaScript based diagramming and charting tool that renders Markdown-inspired text definitions to create and modify diagrams dynamically.
+
+<box type="info">
+
+All supported diagrams and detailed configurations are available in [the Mermaid official documentation](https://mermaid-js.github.io/mermaid/).
+
+</box>
+
+To enable this plugin, add `mermaid` to your site's plugins. Optionally, you can specify an alternative URL to load the Mermaid code in the plugin context.
+
+```js {heading="site.json"}
+{
+  ...
+  "plugins": [
+    "mermaid"
+  ],
+  "pluginsContext": {
+    "mermaid": {
+      "address": "https://unpkg.com/mermaid@10/dist/mermaid.esm.min.mjs" // replace with URL of your choice
```

Comment:
Can add that this is optional? I know you mentioned it above but some might just copy the code wholesale without reading the top

Reply:
Yea. It also gave me the impression that you need to input it but it is actually optional. You can make it more clear that this configuration is only needed if you want to change the mermaid cdn url.

Reply:
Can you address this @Tim-Siu